ros2_batch_job/__main__.py

```diff
@@ -359,18 +359,27 @@ def build_and_test(args, job):
     # xunit2 format is needed to make Jenkins xunit plugin 2.x happy
     with open('pytest.ini', 'w') as ini_file:
         ini_file.write('[pytest]\njunit_family=xunit2')
-    # check if packages have a pytest.ini file and add the xunit2
-    # format if it is not present
+    # check if packages have a pytest.ini file that doesn't choose junit_family=xunit2
+    # and patch configuration if needed to force the xunit2 value
+    pytest_6_or_greater = job.run([
+        '"%s"' % job.python, '-c', '\''
```

Comment:
Nitpick: assuming a double quote should work to use `'"'` instead of `'\''`.

Same for the closing quote.

Reply:
this can not use double quotes since they are being using in 369.

Reply:
Then avoid the escaping with `"'"`.

Reply:
Or use single quotes for the version string in line 369.

Reply:
thanks, I used `" ' "`